eng/Directory.Build.Data.props

```diff
@@ -44,6 +44,7 @@
     <EnableFxCopAnalyzers Condition="'$(IsShippingClientLibrary)' == 'true'">true</EnableFxCopAnalyzers>
     <EnableStyleCopAnalyzers Condition="'$(EnableStyleCopAnalyzers)' == '' and '$(IsClientLibrary)' == 'true'">true</EnableStyleCopAnalyzers>
     <GenerateAPIListing Condition="'$(IsShippingClientLibrary)' == 'true'">true</GenerateAPIListing>
+    <RunApiCompat Condition="'$(IsShippingClientLibrary)' == 'true'">true</RunApiCompat>
```

Comment:
Nit - `EnableBreakingChangeAnalysis` might match the other properties better and avoid confusion with your `RunApiCompat` target that uses this property as a condition.

Reply:
I think "Enable" here would be more consistent as well.